### PR TITLE
Codelabs 4 codelabs

### PR DESCRIPTION
**Pull request by TamimiGitHub**

### CODE REVIEW

File: markdown/codelab-4-codelab/codelab-4-codelab.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 author: Marc DiPasquale
 summary: Create a CodeLab Using Markdown
 id: codelab-4-codelab
-tags: 
+tags:
 categories: codelab,markdown
 environments: Web
-status: Published
+status: Published
```
Comment (TamimiGitHub) [inline markdown/codelab-4-codelab/codelab-4-codelab.md:7]: Remove tab

File: codelabs/codelab-4-codelab/index.html [hunk 2]
```diff
@@ -21,11 +21,11 @@
 </head>
 <body>
   <google-codelab-analytics gaid="UA-3921398-10"></google-codelab-analytics>
-  <google-codelab codelab-gaid="UA-3921398-10"
+  <google-codelab codelab-gaid=""
```
Comment (Mrc0113) [inline codelabs/codelab-4-codelab/index.html:24]: missing GAID

File: codelabs/codelab-4-codelab/index.html [hunk 3]
```diff
@@ -21,11 +21,11 @@
...
                   id="codelab-4-codelab"
                   title="CodeLab to Create a CodeLab"
                   environment="web"
-                  feedback-link="https://github.com/SolaceDev/codelab-4-codelab">
+                  feedback-link="">
```
Comment (Mrc0113) [inline codelabs/codelab-4-codelab/index.html:28]: missing link

File: codelabs/codelab-4-codelab/index.html [hunk 4]
```diff
@@ -233,15 +230,16 @@ <h2 is-upgraded>Stage your Codelab</h2>
...
+<p>Add your changes and any newly created files, then commit &amp; push the changes. From your root directory, execute:</p>
 <pre><code>git add .
 git commit -m &#39;Added or Updated &lt;header-metadata-id&gt; codelab&#39;
-git push --set-upstream origin add-codelab-foo
+git push origin add-codelab-&lt;id&gt;
 </code></pre>
 <h2 is-upgraded>Create a Pull Request</h2>
-<p>Now that your changes have been pushed to a new branch branch we need to request that they get pulled into the master branch to go live on solace.dev/codelabs.</p>
-<p>Do this by navigating to the github repo at: <a href="https://github.com/SolaceDev/solace-dev-codelabs" target="_blank">https://github.com/SolaceDev/solace-dev-codelabs</a></p>
+<p>Now that your changes have been pushed to a new branch, navigate to your fork <code>https://github.com/&lt;Youe_Github_User&gt;/solace-dev-codelabs</code> and create a pull request against master.</p>
```
Comment (Mrc0113) [inline codelabs/codelab-4-codelab/index.html:239]: Youe should be Your

File: markdown/codelab-4-codelab/codelab-4-codelab.md [hunk 5]
```diff
@@ -262,20 +259,22 @@ cd markdown/<header-metadata-id>
...
 git commit -m 'Added or Updated <header-metadata-id> codelab'
-git push --set-upstream origin add-codelab-foo
+git push origin add-codelab-<id>
 ```
 
 ### Create a Pull Request
-Now that your changes have been pushed to a new branch branch we need to request that they get pulled into the master branch to go live on solace.dev/codelabs. 
-
-Do this by navigating to the github repo at: [https://github.com/SolaceDev/solace-dev-codelabs](https://github.com/SolaceDev/solace-dev-codelabs)
+Now that your changes have been pushed to a new branch, navigate to your fork `https://github.com/<Youe_Github_User>/solace-dev-codelabs` and create a pull request against master.
```
Comment (Mrc0113) [inline markdown/codelab-4-codelab/codelab-4-codelab.md:270]: Youe should be Your

**Comment by TamimiGitHub** [comment]
https://github.com/SolaceDev/solace-dev-codelabs-site/pull/38

**Pull request merged**